Makefile

```diff
@@ -1,5 +1,6 @@
 IMAGE_BUILD_CMD ?= "docker"
 REGISTRY_NAMESPACE ?= "ocs-dev"
+REGISTRY_NAME ?= "${REGISTRY_NAME}"
```

Comment:
You need to set a default value here,
```make
REGISTRY_NAME ?= "quay.io"
```

Reply:
done, I had it like this in another branch, but used a vim replacement when moving the change to this branch. Thanks for catching it

Reply:
Still see the same here. This hasn't changed.